lua/kickstart/plugins/debug.lua

```diff
@@ -85,6 +85,11 @@ return {
     dap.listeners.before.event_exited['dapui_config'] = dapui.close
 
     -- Install golang specific config
-    require('dap-go').setup()
+    require('dap-go').setup {
+      delve = {
+        -- On Windows delve must not be run attached or it crashes.
+        detached = vim.loop.os_uname().sysname ~= 'Windows_NT',
```

Comment:
maybe this should be used:
```
vim.fn.has 'win32' == 1
```
which is for example already used in init.lua line 676

Reply:
If that is preferred, please amend the PR (maintainers should have access). I've never worked with the nvim API before so I barely know it.

Reply:
I'm not a maintainer, just occasional contributor :)
It was just a suggestion, this is the first time I see `vim.loop.os_uname`, but I saw `vim.fn.has` many times before, but I see both are used in various plugins, for example if we check in: `~/.local/share/nvim/` :
```
grep -r 'fn.has.*win' . | wc -l
40

grep -r 'vim.loop.os_uname.*Win' . | wc -l
3
```
we can see that both are used but fn.has is more common.
To conclude, it's fine as is. You can disregard my comment.

Reply:
It is valuable input to maintainers 👍 Thank you for taking the time to look at it.

Reply:
Thanks for your contribution!

Please just try the API @dam9000 suggested locally and amend your PR if it works.

It doesn't make any sense for one of us to amend your PR. That's the purpose of review feedback, to help *you* make your contribution better :)

Thanks for understanding.

Reply:
Sure thing. Right after dinner. 🍢